### DISCUSSION TITLE
BE node released accidently

### DISCUSSION THREAD

**Discussion by gilboom** [apache/doris#25335] [A - General / Q&A]
I have three BE node, and every table has three replicas. Ops release one BE node in aws accidently last night. He buy a new node in aws today. Should I drop the released BE node directly by `DROP` action and add the new BE node by `ADD` action ? Will the new node copy data from the old node?

**Comment by yagagagaga** [comment]
You can refer this [page](https://doris.apache.org/docs/dev/admin-manual/cluster-management/elastic-expansion/#add-be-nodes).
